src/components/ui/Question.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import Button from "./Button";
+import { nanoid } from "nanoid";
+import { Heading2 } from "./Typography";
+import classNames from "classnames";
+const Question = ({ question, answers, selectAnswer, id, selected, correct, incorrect_answers, quiz }) => {
+
+
+    return <div className="flex flex-col gap-y-4 border-b-2 border-b-[#DBDEF0] items-start mt-5">
+        <Heading2 children={question} />
+        <div className="flex gap-x-5 mb-4 flex-wrap lg:flex-row gap-y-5">
+            {answers.sort().map((el, index) => {
+                let check = selected === index
+                let text = answers[index];
+                let right = !incorrect_answers.includes(text)
+                console.log(check, quiz)
```

Comment:
The consoles are used for debugging, no need to add them in production, as it will increase the file size for nothing.